hnn_core/cell_response.py

```diff
@@ -6,6 +6,8 @@
 
 from glob import glob
 import numpy as np
+import os
```

Comment:
order of imports is standard libraries first, then numpy/matplotlib etc, then any other libraries, then hnn-core functions.

Reply:
Is this an old comment? The current order seems to match this

Reply:
yeah ...